docs/tutorials/migrate-to-chiselled-rock.rst

```diff
@@ -0,0 +1,268 @@
+- skopeo installed (https://github.com/containers/skopeo)
+- Docker installed (https://snapcraft.io/docker)
+- a text editor
+
+
+Install Rockcraft
+-----------------
+
+Install Rockcraft on your host:
+
+.. literalinclude:: code/hello-world/task.yaml
+    :language: bash
+    :start-after: [docs:install-rockcraft]
+    :end-before: [docs:install-rockcraft-end]
+    :dedent: 2
+
+Project Setup
+-------------
+
+For this tutorial, the reference Docker image will be
+`Microsoft's .NET Runtime 6.0
+<https://github.com/dotnet/dotnet-docker/tree/main/src/runtime/6.0/jammy>`_.
+The target base Ubuntu release will be Jammy, and the target architecture will
+be ``amd64``.
+
+Create a new directory, write the reference Dockerfile (pasted below) into a
+text editor and save it as ``Dockerfile``:
+
+.. literalinclude:: code/migrate-to-chiselled-rock/Dockerfile
+    :language: Docker
+
+For the sake of comparison, start by building this Docker image by running:
+
+
+.. literalinclude:: code/migrate-to-chiselled-rock/task.yaml
+    :language: bash
+    :start-after: [docs:build-docker-image]
+    :end-before: [docs:build-docker-image-end]
+    :dedent: 2
+
+The output should look as follows:
+
+..  code-block:: text
+    :emphasize-lines: 16
+
+    [+] Building 0.6s (10/10) FINISHED
+    => [internal] load .dockerignore0.0s
+    => => transferring context: 2B0.0s
+    => [internal] load build definition from Dockerfile    0.0s
+    => => transferring dockerfile: 881B   0.0s
+    => [internal] load metadata for mcr.microsoft.com/dotnet/runtime-deps:6.0.16-jammy-amd64    0.1s
+    => [internal] load metadata for docker.io/amd64/buildpack-deps:jammy-curl    0.6s
+    => [stage-1 1/3] FROM mcr.microsoft.com/dotnet/runtime-deps:6.0.16-jammy-amd64@sha256:e764c6f0cc866a1f2932  0.0s
+    => [installer 1/2] FROM docker.io/amd64/buildpack-deps:jammy-curl@sha256:e1f00c6daf4cd328bbef9c52e6c60f18a  0.0s
+    => CACHED [installer 2/2] RUN dotnet_version=6.0.16&& curl -fSL --output dotnet.tar.gz https://dotnet  0.0s
+    => CACHED [stage-1 2/3] COPY --from=installer [/dotnet, /usr/share/dotnet]   0.0s
+    => CACHED [stage-1 3/3] RUN ln -s /usr/share/dotnet/dotnet /usr/bin/dotnet   0.0s
+    => exporting to image   0.0s
+    => => exporting layers  0.0s
+    => => writing image sha256:a24cab51d4d02019dafcd22a2e2a3e1e6d033f9bbf1cb401d465cb2426bb2264 0.0s
+    => => naming to docker.io/library/dotnet-runtime:reference   0.0s
+
+Now, inspect this .NET reference image's size and make sure it is functional:
+
+.. literalinclude:: code/migrate-to-chiselled-rock/task.yaml
+    :language: bash
+    :start-after: [docs:inspect-docker-image]
+    :end-before: [docs:inspect-docker-image-end]
+    :dedent: 2
+
+The output should look as follows:
+
+..  code-block:: text
+    :emphasize-lines: 2
+
+    REPOSITORY TAG IMAGE ID CREATED SIZE
+    dotnet-runtime   reference   a24cab51d4d0   4 minutes ago   187MB
+
+    global.json file:
```

Comment:
this is being extra picky but a suggestion: the output of the two commands (image size and running the container) is concatenated here with no indication of which is which. It's fairly obvious but might be confusing for new users

Reply:
will split it